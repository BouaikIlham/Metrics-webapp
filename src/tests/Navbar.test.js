import { render, screen } from '@testing-library/react';
import { BrowserRouter } from 'react-router-dom';
import '@testing-library/jest-dom';
import { Provider } from 'react-redux';
import Navbar from '../Components/Navbar';
import store from '../redux/configStore';

describe('Navbar tests', () => {
  test('should render', () => {
    expect(Navbar).toMatchSnapshot();
  });

  test('Checks if header content is there by checking the logo text', () => {
    render(
      <Provider store={store}>
        <BrowserRouter>
          <Navbar />
        </BrowserRouter>
        ,
      </Provider>,
    );

    expect(
      screen.getByRole('heading', { name: /2022/i }),
    ).toBeInTheDocument();
  });
});
