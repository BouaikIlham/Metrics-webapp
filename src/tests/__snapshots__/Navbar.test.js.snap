// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[` tests should render 1`] = `[Function]`;

exports[`Header tests should render 1`] = `[Function]`;

exports[`Na tests should render 1`] = `[Function]`;

exports[`Navbar tests should render 1`] = `[Function]`;
